pom.xml

```diff
@@ -237,10 +213,6 @@
                     <groupId>com.nimbusds</groupId>
                     <artifactId>nimbus-jose-jwt</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>org.apache.zookeeper</groupId>
-                    <artifactId>zookeeper</artifactId>
-                </exclusion>
```

Comment:
Why do you remove these exclusions, are they required somehow?

Reply:
The dependency which contains this exclusion was updated and I think this inner dependency was updated too because it doesn't contain vulnerabilities anymore, so I removed it from the exclusions list

Reply:
:+1: